src/reporters/remote_reporter.js

```diff
@@ -27,7 +27,7 @@ import NoopMetricFactory from '../metrics/noop/metric_factory';
 const DEFAULT_BUFFER_FLUSH_INTERVAL_MILLIS = 10000;
```

Comment:
is 10s consistent with other languages?  I rather thing 1s default would be preferable.